api/plan.go

```diff
@@ -68,15 +66,43 @@ const (
 	// in turns belongs to the instance.
 	// - Support defining extra data source for a database and exposing the related data source UI.
 	FeatureDataSource FeatureType = "bb.feature.data-source"
+	// FeatureDBAWorkflow enforces the DBA workflow.
+	//
+	// - Developers can't create and view instances since they are exclusively by DBA, they can
+	//   only access database.
+	// - Developers can submit troubleshooting issue.
+	FeatureDBAWorkflow FeatureType = "bb.feature.dba-workflow"
+	// FeatureLGTM checks LGTM comments.
+	FeatureLGTM FeatureType = "bb.feature.lgtm"
+	// FeatureMultiTenancy allows user to enable tenant mode for the project.
+	//
+	// Tenant mode allows user to track a group of homogeneous database changes together.
+	// e.g. A game studio may deploy many servers, each server is fully isolated with its
+	// own database. When a new game version is released, it may require to upgrade the
+	// underlying database schema, then tenant mode will help the studio to track the
+	// schema change across all databases.
+	FeatureMultiTenancy FeatureType = "bb.feature.multi-tenancy"
+	// FeatureMultiTenancy allows user to perform online-migration.
+	FeatureOnlineMigration FeatureType = "bb.feature.online-migration"
```

Comment:
exported: comment on exported const FeatureOnlineMigration should be of the form "FeatureOnlineMigration ..." (revive)